test_fms/mpp/test_mpp_gatscat.F90

```diff
@@ -34,7 +34,9 @@ program test_mpp_gatscat
   use mpp_mod, only : mpp_sync
   use mpp_mod, only : mpp_declare_pelist, mpp_set_current_pelist, mpp_set_stack_size
   use mpp_mod, only : mpp_gather, mpp_scatter, mpp_error, FATAL
-  use mpp_io_mod, only: mpp_io_init, mpp_flush
+#ifdef use_deprecated_io  
+  use mpp_io_mod, only: mpp_flush
```

Comment:
you can delete this use and any calls to mpp flush if there is any

Reply:
@mcallic2 could you remove these `mpp_flush` statements for this and the other test? I don't think they are even used and it'd be better to not use the old io at all in unrelated tests